tests/covalent_dispatcher_tests/_core/execution_test.py

```diff
@@ -317,27 +339,42 @@ def workflow(x):
         res1 = ct.electron(sublattice_task(x), executor=dask_exec)
         return res1
 
+    asyncio.set_event_loop(event_loop)
+
```

Comment:
Here as well, I think the event loop will already be set by pytest unless I'm missing something here.